cheroot/connections.py

```diff
@@ -84,9 +100,15 @@ def put(self, conn):
         if conn.rfile.has_data():
             self._readable_conns.append(conn)
         else:
-            self._selector.register(
-                conn.socket.fileno(), selectors.EVENT_READ, data=conn,
-            )
+            self._put_q.append(conn)
+            with self._ctrl_lock:
+                self._ctrl_tx.send(self._CTRL_MSG_PUT)
+
+    def _get_selector_conns(self):
+        # helper to retrieve conns registered with the selector
```

Comment:
I think this goes better as a docstring
```suggestion
        """Retrieve connections registered with the selector."""
```